qiskit/circuit/quantumcircuit.py

```diff
@@ -2265,17 +2265,98 @@ def reset(self, qubit: QubitSpecifier) -> InstructionSet:
         return self.append(Reset(), [qubit], [])
 
     def measure(self, qubit: QubitSpecifier, cbit: ClbitSpecifier) -> InstructionSet:
-        """Measure quantum bit into classical bit (tuples).
+        """Measure quantum bit (qubit) into classical bit (cbit).
+
+        When a qubit is measured, its state collapses to a classical bit and copied to a
+        classical wire.
 
         Args:
-            qubit: qubit to measure.
-            cbit: classical bit to place the measurement in.
+            qubit: qubit/s to measure.
+            cbit: classical bit/s to place the measurement in.
 
         Returns:
             qiskit.circuit.InstructionSet: handle to the added instructions.
 
         Raises:
             CircuitError: if arguments have bad format.
+
+        Examples:
+            In this example, a qubit is measured and the result of that measurement is stored in the
+            classical bit (usually expressed in diagrams as a double line):
+
+            .. code-block::
+
+               from qiskit import QuantumCircuit
+               circuit = QuantumCircuit(1,1)
+               circuit.h(0)
+               circuit.measure(0, 0)
+               circuit.draw()
+
+
+            .. parsed-literal::
+
+                      ┌───┐┌─┐
+                   q: ┤ H ├┤M├
+                      └───┘└╥┘
+                 c: 1/══════╩═
+                            0
+
+            It is possible to call ``measure`` with lists of qubits and cbits of the same size:
+
+            .. code-block::
+
+               circuit = QuantumCircuit(2,2)
+               circuit.measure([0,1], [0,1])  # same as "circuit.measure(0,0); circuit.measure(1,1);"
+
+
+            .. parsed-literal::
+
+                      ┌─┐
+                 q_0: ┤M├───
+                      └╥┘┌─┐
+                 q_1: ─╫─┤M├
+                       ║ └╥┘
+                 c: 2/═╩══╩═
+                       0  1
```

Comment:
The example here wouldn't actually produce the given output, because it doesn't have a `draw` (not sure how precise you care about being, though).

I'd maybe change the comment to be a proper construction of a second circuit, with the comment explaining "these two forms produce identical output", just to be a bit clearer (and not to have long code lines in the rendered docs).

Reply:
If I understood you correctly, I moved away from the drawings into an equivalence approach in 1a474f66f22bb0d3cc9763a8fc216bb4f87b8020